db/migrate/20210616203622_add_threshold_hash_to_jurisdictions.rb

```diff
@@ -2,9 +2,20 @@ class AddThresholdHashToJurisdictions < ActiveRecord::Migration[6.1]
   def up
     add_column :jurisdictions, :current_threshold_condition_hash, :string
 
-    # populate :current_threshold_condition_hash
-    Jurisdiction.all.find_each do |jur|
-      jur.update(current_threshold_condition_hash: Digest::SHA256.hexdigest(jur[:path] + ThresholdCondition.where(jurisdiction_id: jur.path_ids).size.to_s))
+    # Run admin:import_or_update_jurisdictions task without checks to ensure that threshold condition with :current_threshold_condition_hash exists
+    ActiveRecord::Base.transaction do
+      config_contents = YAML.load_file('config/sara/jurisdictions.yml')
+
+      config_contents.each do |jur_name, jur_values|
+        parse_jurisdiction(nil, jur_name, jur_values)
```

Comment:
Why is it required to parse each jurisdiction to update the current hash?

Reply:
@mayerm94 brought up a concern where the current threshold condition hash might not be valid (unlikely but still possible) and so by parsing the jurisdictions here, we’re guaranteed to not run into that edge case

Reply:
@mayerm94 can you outline the reasoning? Just so I'm aware in the future